### PR TITLE
Implemented issue #674 (Include swapin/swapout metric in memory)

### PR DESCRIPTION
**Pull request by ccztux**
Implemented: #674 Include swapin/swapout metric in memory.

### CODE REVIEW

File: agent/listener/psapi.py [hunk 1]
```diff
@@ -149,8 +149,20 @@ def get_memory_node():
     mem_swap_percent = RunnableNode('percent', method=lambda: (ps.swap_memory().percent, '%'))
     mem_swap_used = RunnableNode('used', method=lambda: (ps.swap_memory().used, 'B'))
     mem_swap_free = RunnableNode('free', method=lambda: (ps.swap_memory().free, 'B'))
+    node_children = [mem_swap_total, mem_swap_free, mem_swap_percent, mem_swap_used]
+
+    # Unix specific metrics ~ sorry Windows! :'(
+    if environment.SYSTEM != 'Windows':
+        try:
```
Comment (ccztux) [inline agent/listener/psapi.py:156]: As descibed in the psutil [documentation](https://psutil.readthedocs.io/en/latest/#psutil.swap_memory) on windows sin and sout on Windows are always set to ```0```. I have excluded sin and sout for Windows systems.
@jomann09 How do you think about that? Should sin and sout available on Windows systems?

File: agent/listener/psapi.py [hunk 2]
```diff
@@ -149,8 +149,20 @@ def get_memory_node():
     mem_swap_percent = RunnableNode('percent', method=lambda: (ps.swap_memory().percent, '%'))
     mem_swap_used = RunnableNode('used', method=lambda: (ps.swap_memory().used, 'B'))
     mem_swap_free = RunnableNode('free', method=lambda: (ps.swap_memory().free, 'B'))
+    node_children = [mem_swap_total, mem_swap_free, mem_swap_percent, mem_swap_used]
+
+    # Unix specific metrics ~ sorry Windows! :'(
+    if environment.SYSTEM != 'Windows':
```
Comment (ccztux) [inline agent/listener/psapi.py:155]: As descibed in the psutil [documentation](https://psutil.readthedocs.io/en/latest/#psutil.swap_memory) on windows sin and sout on Windows are always set to ```0```. I have excluded sin and sout for Windows systems.
@jomann09 How do you think about that? Should sin and sout available on Windows systems?

File: agent/listener/psapi.py [hunk 3]
```diff
@@ -149,8 +149,20 @@ def get_memory_node():
     mem_swap_percent = RunnableNode('percent', method=lambda: (ps.swap_memory().percent, '%'))
     mem_swap_used = RunnableNode('used', method=lambda: (ps.swap_memory().used, 'B'))
     mem_swap_free = RunnableNode('free', method=lambda: (ps.swap_memory().free, 'B'))
+    node_children = [mem_swap_total, mem_swap_free, mem_swap_percent, mem_swap_used]
+
+    # Unix specific metrics ~ sorry Windows! :'(
```
Comment (ccztux) [inline agent/listener/psapi.py:154]: As descibed in the psutil [documentation](https://psutil.readthedocs.io/en/latest/#psutil.swap_memory) on windows sin and sout on Windows are always set to ```0```. I have excluded sin and sout for Windows systems.
@jomann09 How do you think about that? Should sin and sout available on Windows systems?

File: agent/listener/psapi.py (hunk 2 above)
Comment (ccztux) [inline agent/listener/psapi.py:155]: As descibed in the psutil [documentation](https://psutil.readthedocs.io/en/latest/#psutil.swap_memory) on windows sin and sout on Windows are always set to ```0```. I have excluded sin and sout for Windows systems.
@jomann09 How do you think about that? Should sin and sout available on Windows systems?

**Comment by ccztux** [comment]
This pull request breaks the actual swap checks:
```
[root@centos7_ncpa_build client]# ./check_ncpa.py -H 10.0.0.76 -t mytoken -M 'memory/swap'
OK: Used swap was 0.00 % (Used: 0.00 GiB, Free: 1.50 GiB, Swapped_out: 0.00 GiB, Swapped_in: 0.00 GiB, Total: 1.50 GiB) | 'used'=0.00GiB;;; 'free'=1.50GiB;;; 'swapped_out'=0.00GiB;;; 'swapped_in'=0.00GiB;;; 'total'=1.50GiB;;;
```

```
{
    "swap": {
        "used": [
            0,
            "GiB"
        ],
        "swapped_out": [
            0,
            "GiB"
        ],
        "swapped_in": [
            0,
            "GiB"
        ],
        "total": [
            1.5,
            "GiB"
        ],
        "percent": [
            0,
            "%"
        ],
        "free": [
            1.5,
            "GiB"
        ]
    }
}
```

I am not sure, where to put the new metrics in the api, without breaking it and where it makes sense.

**Comment by ccztux** [comment]
Now the original swap check works as before:
```
[root@centos7_ncpa_build client]# ./check_ncpa.py -H 10.0.0.76 -t mytoken -M 'memory/swap'
OK: Used swap was 0.00 % (Used: 0.00 GiB, Free: 1.50 GiB, Total: 1.50 GiB) | 'used'=0.00GiB;;; 'free'=1.50GiB;;; 'total'=1.50GiB;;;
```

Changes in the api structure:
```
{
    "swap": {
        "used": [
            0.0, 
            "GiB"
        ], 
        "percent": [
            0.0, 
            "%"
        ], 
        "free": [
            1.5, 
            "GiB"
        ], 
        "swapped_out": [
            0.0, 
            "GiB"
        ], 
        "swapped_in": [
            0.0, 
            "GiB"
        ], 
        "total": [
            1.5, 
            "GiB"
        ]
    }
}
```

**Comment by ccztux** [comment]
@jomann09 I think this is ready to be merged. Please let me know, if you would like to see something to be changed.

**Comment by jomann09** [comment]
This looks good, merging it now.

**Pull request merged**